packages/main/src/themes/Input.css

```diff
@@ -430,12 +430,28 @@
 	box-shadow: var(--_ui5_input_success_icon_box_shadow);
 }
 
+.ui5-input-clear-icon-wrapper {
+	height: var(--_ui5_input_icon_wrapper_height);
+	padding: 0;
+	width: var(--_ui5_input_icon_width);
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	box-sizing: border-box;
+}
 
+:host([value-state]:not([value-state="None"])) .ui5-input-clear-icon-wrapper {
```

Comment:
The issue still exists. Now, it is even worse, because the icon overlap 2px of the input field when in the previous patch set it was 1px.
Following, the way of your implementation, I think that the height of the icon has to be calc(100% - 2px) when the value state is Error, Warning or Information